public/index.php

```diff
@@ -8,5 +8,7 @@
 
 $PDO = Kernel::getDatabase();
 
+$User1 = Kernel::getUser();
```

Comment:
do usunięcia

Reply:
pobieranie użytkownika metodą User::find(1); gdzie 1 to jest ID użytkownika a ty pobierasz Kernel::getUser()